src/Sulu/Component/Content/Document/Subscriber/StructureSubscriber.php

```diff
@@ -258,17 +258,13 @@ private function mapContentToNode($document, NodeInterface $node, $locale)
             $legacyProperty = $this->legacyPropertyFactory->createTranslatedProperty($structureProperty, $locale);
             $legacyProperty->setValue($value);
 
-            try {
-                $contentType->remove(
-                    $node,
-                    $legacyProperty,
-                    null,
-                    $webspaceName,
-                    $locale,
-                    null
-                );
-            } catch (\Exception $e) {
-            }
+            $contentType->remove(
+                $node,
+                $legacyProperty,
+                $webspaceName,
+                $locale,
+                null
+            );
```

Comment:
Tried to remove a property by hand in the phpcr shell, and saved a page again, was still working. So I think we really do not need to check any exceptions here. /cc @wachterjohannes @chirimoya